core/src/epicli/data/common/ansible/playbooks/roles/postgres_exporter/handlers/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: restart postgres-exporter
+  service:
+    name: postgres-exporter
+    state: restarted
+  retries: 10
```

Comment:
There is no "until" key, it will not retry.

Reply:
Added:
until: result is success

Reply:
`register: result` is also needed, these things come in pairs
```
register: result
until: result is success
```

Reply:
done